source/software/debuggers/index.rst

```diff
@@ -0,0 +1,281 @@
+
+Debuggers
+=========
+
+
+Debugging with DDT
+------------------
+
+Linaro DDT (formerly ARM DDT) is a powerful, easy to use graphical debugger. With Linaro DDT it is possible to debug: 
+
+- Single process and multithreaded software 
+- Parallel (MPI) Software 
+- OpenMP
+- Hybrid codes mixing paradigms such as MPI + OpenMP, or MPI + CUDA
+
+
+Linaro Forge supports:
+
+- C, C++, and all derivatives of Fortran, including Fortran 90
+- Python (CPython 2.7), limited support 
+- Parallel languages/models including MPI, UPCm, Fortran 2008 Co-arrays. 
+
+Linaro DDT helps you to find and fix problems on a single thread or across hundreds of thousands of threads. It includes static analysis to highlight potential code problems, integrated memory debugging to identify reads and writes that are outside of array bounds, and integration with MPI message queues. 
+
+Using DDT
+---------
+
+.. tab-set::
```

Comment:
Is this formatted correctly?  From the GitHub viewer, at least, it looks like the tabs aren't rendering, and this is all rendered literally --> https://github.com/NOAA-RDHPCS/noaa-rdhpcs.github.io/blob/7372fcd6892c934e153b7709cd50eca4c7ecda1e/source/software/debuggers/index.rst